macapp/App/Tests/AppTests/OnboardingFeatureTests.swift

```diff
@@ -1478,273 +1477,20 @@ final class OnboardingFeatureTests: XCTestCase {
     ])
   }
 
-  @MainActor
```

Comment:
all these were just moved into the dedicated test file for FDA upgrade onboarding